### PR TITLE
CC-29246 Styling store switcher

### PR DESCRIPTION
**Pull request by andrew-maslov**
Ticket: https://spryker.atlassian.net/browse/CC-29246

### CODE REVIEW

File: src/Pyz/Yves/ShopUi/Theme/default/components/organisms/navigation-top/navigation-top.twig [hunk 1]
```diff
@@ -54,9 +54,17 @@
             {% widget 'LanguageSwitcherWidget' args [app.request.pathInfo, app.request.queryString, app.request.requestUri] only %}{% endwidget %}
         </li>
 
-        <li class="col {{ config.name }}__action-item">
-            {% widget 'StoreSwitcher' only %}{% endwidget %}
-        </li>
+        {% widget 'StoreSwitcher' with {
+            embed:{
+                configName: config.name,
+            }
```
Comment (danilmoro3) [inline src/Pyz/Yves/ShopUi/Theme/default/components/organisms/navigation-top/navigation-top.twig:60]: Please fix format

File: src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant/variant.twig [hunk 2]
```diff
@@ -1,62 +1,39 @@
...
-                class: 'name',
+                data: {
+                    options: options,
+                },
                 attributes: {
                     name: data.formName,
+                    'data-form-submitter': true,
                     'config-width': '100%',
                     'config-theme': data.selectTheme,
-                    'auto-init': data.autoInitSelect,
```
Comment (danilmoro3) [inline src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant/variant.twig:36]: Does molecule work correctly with this change?

**Review by danilmoro3 — APPROVED** [review]

**Review by danilmoro3 — APPROVED** [review]

File: src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant/variant.twig (hunk 2 above)
Comment (andrew-maslov) [reply src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant/variant.twig:36]: Yes, this attribute should be removed.
It already removed on [b2b demoshop](https://github.com/spryker/b2b-demo-shop-internal/pull/918)

Comment (FRITS-Kh) [reply src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant/variant.twig:36]: So please apply changes from the PR.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.ts [hunk 3]
```diff
@@ -23,7 +18,7 @@ export default class SlickCarousel extends Component {
     protected mapEvents(): void {
         $(this.slider).on('init', async () => {
             this.showSlider();
-            await mount();
```
Comment (FRITS-Kh) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.ts:26]: `async` should have `await`, I guess `await` should be added in the `b2b` repo.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — APPROVED** [review]

**Pull request merged**